tests/checks/core/test_certificate_expiration.py

```diff
@@ -57,6 +57,20 @@ async def test_negative():
     assert data == {"expires": next_month.isoformat()}
 
 
+async def test_positive_bounded_maximum():
+    url = "https://fake.local"
+
+    last_year = utcnow() - timedelta(days=2365)
+    next_month = utcnow() + timedelta(days=15)
```

Comment:
Very much a nitpick, but this variable name is only accurate for the 2nd half of any given month. Do we actually want this date to be "a date in the next calendar month", or do we just want the date to be 15 days from now?